### DISCUSSION TITLE
is there a way to get list of forwarded ports by go-ios?

### DISCUSSION THREAD

**Discussion by mega6453** [danielpaulus/go-ios#372] [Q&A] [answered]
In adb, we have a command "adb  forward --list" which will list out all the forwarded ports by adb. Similarly do we have in go-ios when we forward a port ?

**Comment by mega6453** [comment]
@danielpaulus

**Comment by nanoscopic** [comment, marked as answer]
When go-ios ( or similar tools such as pmd3 etc ) forward ports they just initiate a protocol with a request to forward the port you selected. When you close the port forwarding process that ends. You could use 'ps' and analyze the running processes you have started to look at the cli options.

Or patch go-ios to keep tracking of all the instances currently forwarding ports somewhere.

To my current awareness there is no api access via lockdown/rsd to list "currently forwarded port processes", hence it isn't implemented.
